streamz/dask.py

```diff
@@ -140,6 +157,24 @@ def update(self, x, who=None):
         return self._emit(result)
 
 
+@DaskStream.register_api()
+class filter(DaskStream):
```

Comment:
I think you also need the modifications to the gather and other nodes as well.

Reply:
I made changes to gather already. I also compared other nodes. There is a slightly difference between the dask.starmap and parallel.starmap. Do I need to change that one?

Reply:
I made changes to gather already. I also compared other nodes. There is a slightly difference between the dask.starmap and parallel.starmap. Do I need to change that one?